### PR TITLE
chore(helm): Update README, add StorageClass parameters

### PR DESCRIPTION
**Pull request by niladrih**
Signed-off-by: Niladri Halder <niladri.halder@mayadata.io>

This PR:
1. Updates README
2. Add CustomServerConfig and isDefaultClass parameters

### CODE REVIEW

File: deploy/helm/charts/README.md [hunk 1]
```diff
@@ -28,11 +28,13 @@ _See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation
 
 ## Install Chart
 
-Please visit the [link](https://openebs.github.io/dynamic-nfs-provisioner/) for install instructions via helm3.
+The chart requires a StorageClass to provision the backend volume for the NFS share. If a StorageClass is not specified (use `--set-string nfsStorageClass.backendStorageClass` to specify -- details given below), the default StorageClass is used.
+
+Please visit this [link](https://openebs.github.io/dynamic-nfs-provisioner/) for helm 3 installation instructions.
```
Comment (kmova) [inline deploy/helm/charts/README.md]:
```suggestion
Please visit this [link](https://helm.sh/docs/) for helm 3 installation instructions.
```

Comment (kmova) [reply deploy/helm/charts/README.md]: Probably another step is required to add openebs-nfs repo. 

```console
# add openebs nfs repo
helm repo add openebs-nfs https://openebs.github.io/dynamic-nfs-provisioner/
```

File: deploy/helm/charts/README.md [hunk 2]
```diff
@@ -55,39 +57,53 @@ _See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command doc
 
 ```console
 # Helm
-$ helm upgrade [RELEASE_NAME] [CHART] --install
+helm upgrade [RELEASE_NAME] [CHART] --install --namespace [NAMESPACE]
```
Comment (kmova) [inline deploy/helm/charts/README.md:91]: q_: do we need to provide the --install in upgrade command as well?

Comment (niladrih) [reply deploy/helm/charts/README.md:60]: Nope. Removing it.

Comment (niladrih) [reply deploy/helm/charts/README.md:91]: We don't need it. Keeping it as per discussion.

**Review by kmova — APPROVED** [review]

**Pull request merged**